<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kao_heClass</class>
 <widget class="QMainWindow" name="Kao_heClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kao_he</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>400</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>欢 迎 来 到 抽 奖 环 节 ~</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="jin">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>进 入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chou">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>320</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>抽 奖</string>
    </property>
   </widget>
   <widget class="QLabel" name="haoyun">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>300</width>
      <height>300</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_1">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2-66          </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2-72</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_4">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3-30</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>2-76   </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_6">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3-32</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_5">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>3-31</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-43</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>190</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-39</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_10">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-56</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>220</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-47</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-59</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLabel" name="t_11">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>4-58</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="tip">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>幼圆</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>haoyun</zorder>
   <zorder>label</zorder>
   <zorder>jin</zorder>
   <zorder>chou</zorder>
   <zorder>t_1</zorder>
   <zorder>t_2</zorder>
   <zorder>t_4</zorder>
   <zorder>t_3</zorder>
   <zorder>t_6</zorder>
   <zorder>t_5</zorder>
   <zorder>t_8</zorder>
   <zorder>t_7</zorder>
   <zorder>t_10</zorder>
   <zorder>t_9</zorder>
   <zorder>t_12</zorder>
   <zorder>t_11</zorder>
   <zorder>tip</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Kao_he.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chou</sender>
   <signal>clicked()</signal>
   <receiver>Kao_heClass</receiver>
   <slot>chou_ti()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>395</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>386</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jin</sender>
   <signal>clicked()</signal>
   <receiver>Kao_heClass</receiver>
   <slot>jin_chuang()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>514</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>chou_ti()</slot>
  <slot>jin_chuang()</slot>
 </slots>
</ui>
